client/app/components/ApplicationArea/ApplicationLayout/VersionInfo.jsx

```diff
@@ -15,7 +15,7 @@ export default function VersionInfo() {
           {/* eslint-disable react/jsx-no-target-blank */}
           <Link href="https://version.redash.io/" className="update-available" target="_blank" rel="noopener">
             Update Available <i className="fa fa-external-link m-l-5" aria-hidden="true" />
-            <span className="sr-only">&#40;External link&#41;</span>
+            <span className="sr-only">{"(External link)"}</span>
```

Comment:
Sorry, I may have been not that clear since I also put the quotes, but what I mean is: does this needs this at all? Can't it be just `<span className="sr-only">(External link)</span>`?

Reply:
No, it can't. I changed back bc I thought this way was clearer.

Reply:
Why it can't?  HTML/jsx wise there's nothing that holds you from writing it directly (we do it in several other places in the codebase). As for screen-readers I don't really see how having it escaped would make a difference 🤔

I'm just curious because this really looks weird to me

Reply:
For some reason it didn't work on my initial tests but now it seems fine. Thanks for pointing this, just reverted it.